app/sesame/automod/vocab.go

```
@@ -0,0 +1,235 @@
+	// dominates: the audit's FP corpus was dominated by one-off reaction spam
+	// that would each clear a low bar instantly.
+	vocabTau = 20
+
+	// vocabSenders d=8 distinct senders required alongside vocabTau. Chosen so
+	// laundering needs 8 colluding accounts per token per channel; smaller d
+	// was trivially brigaded in the audit's raid-shaped traffic.
+	vocabSenders = 8
+
+	// vocabChanCap matches bot_stats.go channelStatsMaxKeys discipline (see
+	// baselineChanCap): stalest-half eviction on overflow.
+	vocabChanCap = 4096
```

golangci-lint (2.12.2)

[error] const vocabChanCap is unused

(unused)